Modules/Core/Common/include/itkSmartPointer.h

```diff
@@ -134,6 +134,15 @@ class SmartPointer
     return m_Pointer;
   }
 
+  /** Returns the stored (raw) pointer. Equivalent to `GetPointer()`, but then following the Standard C++ Library naming
+   * conversion (like `std::shared_ptr::get()`). */
+  ObjectType *
+  get() const noexcept
```

Comment:
Why `get` instead of `Get`?

Reply:
Because STL uses `get` (lowercase): https://en.cppreference.com/w/cpp/memory/shared_ptr/get